doc/api/process.md

```diff
@@ -2260,6 +2260,29 @@ process.kill(process.pid, 'SIGHUP');
 When `SIGUSR1` is received by a Node.js process, Node.js will start the
 debugger. See [Signal Events][].
 
+## `process.loadEnvFile(path)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `path` {string | URL | Buffer | undefined}. **Default:** `'./.env'`
```

Comment:
If you look at the code, you can see it is `'.env'`, there's no call to `process.cwd()`.

IMO it makes sense to have a shortcut for `./.env`, as it's a very usual pattern.

Reply:
I agree with the `./.env` shortcut as well, since it's pretty common path and it's default value for `dotenv` package too.

Reply:
> If you look at the code, you can see it is `'.env'`, there’s no call to `process.cwd()`.

If it’s just `'.env'`, not `'./.env'`, then what does it resolve to? Relative to the file or relative to the process?

Reply:
Why would you assume that when all the other FS related APIs do not work like that?

Reply:
FWIW [`process.chdir()` documentation](https://nodejs.org/api/process.html#processchdirdirectory) does not bother to specify that – the [`Worker` constructor documentation](https://nodejs.org/api/worker_threads.html#new-workerfilename-options) does, but also it has more restrictions. IMO it's not necessary to specify, I expect our users are used to use `path.join(__dirname, ...` or `new URL` when specifying a path anyway, but I guess it can't harm.

Reply:
I can’t think of anywhere else we load an _implicit_ file, where the user isn’t required pass in a path. I think it can’t hurt to be more explicit here. Personally I’m always frustrated by `new Worker` not working relative to the current file. When I see `./` I always assume that means “relative to the current file” and not relative to something else. If you look at the start of this thread that was my assumption the entire time, and we got quite deep before either of us realized that we had differing assumptions. If that can happen to us, surely it can happen to at least some portion of our users.

Reply:
> When I see `./` I always assume that means “relative to the current file” and not relative to something else. If you look at the start of this thread that was my assumption the entire time, and we got quite deep before either of us realized that we had differing assumptions. If that can happen to us, surely it can happen to at least some portion of our users.

In my experience, that knowledge comes in like the first weeks of toying with Node.js APIs. I have no idea how you dodge it for so long 🤷‍♂️ And it's not just in Node.js internals btw, it's the case everywhere in the ecosystem.

> Personally I’m always frustrated by `new Worker` not working relative to the current file. When I see `./` I always assume that means “relative to the current file” and not relative to something else.

We had the same discussion about `module.register`, I don't know what you imagine the implementation would look like otherwise. `require` and `import` are the exception, not the rule.

Anyway, I think it's OK to explicitly document that the `path` argument will be interpreted as defined in `fs.md#file-paths`.